src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -3069,6 +3069,14 @@ theorem not_neg (x : BitVec w) : ~~~(-x) = x + -1#w := by
         show (_ - x.toNat) % _ = _ by rw [Nat.mod_eq_of_lt (by omega)]]
       omega
 
+theorem neg_add {x y : BitVec w} : - (x + y) = - x - y := by
+ apply eq_of_toInt_eq
```

Comment:
is indentation correct? seems smaller than the others

Reply:
Thanks, it was indeed shorter than usual!